skore-ui/src/components/ProjectViewCard.vue

```diff
@@ -20,16 +38,35 @@ const emit = defineEmits<{
         <div class="title">{{ props.title }}</div>
         <div class="subtitle" v-if="props.subtitle">
           {{ props.subtitle }}
+          <Transition name="fade">
+            <span v-if="!isLatestUpdate" class="warning">
+              <i class="icon-warning"></i>
+              You are viewing an old update
```

Comment:
About "You are viewing an old update": I know we said "version" should be kept for a later feature, but "an old update" sounds very weird. An update is new, by definition (it's "up to date", "à jour")

Reply:
Could you suggest some better wording ? Maybe @MarieS-WiMLDS & @tuscland have ideas ?

Reply:
Some suggestions:

- "You are viewing historical content" (my preferred)
- "You are viewing historical changes"
- "You are viewing past content"
- "You are viewing archived content"
- "You are viewing previous content"

I'm interested to know your feelings @augustebaum @MarieS-WiMLDS

Reply:
In my opinion all of these wordings are strange, like we are deliberately avoiding the word "versions". Can you explain again why "versions" is not an option here?

Reply:
It was mostly a defensive move that does not make a lot of sense after all. The feature I have in mind could use another name, so let's ditch this artificial constraint.

What about "You are viewing an older version"?

Reply:
I'd prefer "you are viewing an old version" rather than "older", because it might spur the question "older than what?"

Reply:
I prefer "you are viewing an old version" :)

Reply:
"you are viewing an old version" sounds perfect to me

Reply:
![Screenshot 2024-11-06 at 15 44 39](https://github.com/user-attachments/assets/9327363f-5813-40b3-93ac-3d3d58b6438e)

The wording has been changed and the icon is a new one without the circle around it.